android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -608,6 +608,36 @@ public boolean peerConnectionRemoveTrack(int id, String senderId) {
         }
     }
 
+    @ReactMethod
+    public void peerConnectionSendDTMF(String tone, int duration, int interToneGap, int objectID) {
+        ThreadUtils.runOnExecutor(() -> {
+            try {
+                RtpSender audioSender = null;
+                PeerConnection peerConnection = getPeerConnection(objectID);
+                if (peerConnection == null) {
+                    Log.d(TAG, "peerConnectionSendDTMF() peerConnection is null");
+                    return;
+                }
+                
+                for (RtpSender sender : peerConnection.getSenders()) {
+                    if (sender.track().kind().equals("audio")) {
+                        audioSender = sender;
+                    }
+                }
+
+                if (audioSender != null) {
+                    DtmfSender dtmfSender = audioSender.dtmf();
```

Comment:
Can this be null?

Reply:
Not sure, but i think it's better to check for null in case no audioSender found